SessionMessagingKit/Messages/Control Messages/ClosedGroupControlMessage.swift

```diff
@@ -112,7 +112,8 @@ public final class ClosedGroupControlMessage : ControlMessage {
                 let encryptionKeyPair = coder.decodeObject(forKey: "encryptionKeyPair") as? ECKeyPair,
                 let members = coder.decodeObject(forKey: "members") as? [Data],
                 let admins = coder.decodeObject(forKey: "admins") as? [Data] else { return nil }
-            self.kind = .new(publicKey: publicKey, name: name, encryptionKeyPair: encryptionKeyPair, members: members, admins: admins)
+                let expireTimer = coder.decodeObject(forKey: "expireTimer") as! UInt32
```

Comment:
This will crash if there's a closed group control message of type new that doesn't yet have this set